src/stan/services/init/initialize_state.hpp

```diff
@@ -219,20 +212,23 @@ namespace stan {
              ++num_init_tries) {
           for (int i = 0; i < cont_params.size(); ++i)
             cont_params(i) = init_rng();
-          if (initialize_state_values(cont_params, model, output))
+          if (initialize_state_values(cont_params, model, writer))
             break;
         }
 
         if (num_init_tries > MAX_INIT_TRIES) {
-          if (output)
-            *output << std::endl << std::endl
-            << "Initialization between (" << -R << ", " << R
-            << ") failed after "
-            << MAX_INIT_TRIES << " attempts. " << std::endl
-            << " Try specifying initial values,"
-            << " reducing ranges of constrained values,"
-            << " or reparameterizing the model."
-            << std::endl;
+          std::stringstream R_ss, MAX_INIT_TRIES_ss;
```

Comment:
Agreed. I think we can do a better job of printing NaN and Inf on Windows. And control the output so it's more uniform across platforms. I think we should open up a new issues for this topic.